### PR TITLE
[FIX]: Search products by skus error

### LINKED ISSUE #6181: Search products by skus error
Merchant is not able to search products by `skus`.

```
const { search} = useProduct()
search({ skus })
```

Error:

```
ERROR  [VSF][error] [GraphQL error]: Message: Malformed parameter: where: Syntax error while parsing 'where'. Unexpected end of input, expected not or conditional expression (line 1, column 1):                11:47:17

^, Location: [column: 3, line: 73], Path: products 
```

### PR DESCRIPTION
**Pull request by Razz21**
### Related Issues

closes #6181 

### Short Description of the PR

Fixes error when using `skus` to search products in `useProduct` composable
```
ERROR  [VSF][error] [GraphQL error]: Message: Malformed parameter: where: Syntax error while parsing 'where'. Unexpected end of input, expected not or conditional expression (line 1, column 1):                11:47:17

^, Location: [column: 3, line: 73], Path: products 
```

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- I tested manually my code and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme
- [ ] I have written test cases for my code

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

File: packages/core/docs/commercetools/changelog/6181.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  description: 'Fixes useProduct search by skus',
```
Comment (filipsobol) [inline packages/core/docs/commercetools/changelog/6181.js:2]:
```suggestion
  description: 'Fix error thrown by the `search` method in `useProduct` composable when searching by product SKUs',
```

**Review by filipsobol — APPROVED** [review]

File: packages/commercetools/api-client/src/helpers/search/index.ts [hunk 2]
```diff
@@ -77,7 +77,7 @@ const buildProductWhere = (settings: Config, search: ProductWhereSearch) => {
     predicates.push(`id in ("${search.ids.join('","')}")`);
   }
```
Comment (alefbarbeli) [inline packages/commercetools/api-client/src/helpers/search/index.ts:78]:
```suggestion
  if (search?.sku) {
    predicates.push(`sku="${search.sku}"`);
  }
```

From what I've understood from #6181 I think we need to add this rule

**Review by alefbarbeli — CHANGES_REQUESTED** [review]

File: packages/commercetools/api-client/src/helpers/search/index.ts (hunk 2 above)
Comment (alefbarbeli) [reply packages/commercetools/api-client/src/helpers/search/index.ts:78]: But please test if this makes sense and if is the best way.

Comment (Razz21) [reply packages/commercetools/api-client/src/helpers/search/index.ts:78]: @alefbarbeli `search.skus` are passed directly to gql query as variable, 
https://github.com/vuestorefront/vue-storefront/blob/1d0e68c437ecbee664d431b93f730c0bde228485/packages/commercetools/api-client/src/api/getProduct/index.ts#L17

This Error is caused by gql `where` variable passed to query as empty string

Comment (alefbarbeli) [reply packages/commercetools/api-client/src/helpers/search/index.ts:78]: Understood! Well done

**Review by alefbarbeli — APPROVED** [review]

**Pull request merged**